pom.xml

```diff
@@ -16,53 +28,54 @@
 
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>3.8.1</version>
-      <scope>test</scope>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>3.8.1</version>
+        <scope>test</scope>
     </dependency>
 
- <dependency>
-   <groupId>com.arangodb</groupId>
-   <artifactId>arangodb-java-driver</artifactId>
-   <version>4.3.0</version>
- </dependency>
- 
- <dependency>
-			<groupId>redis.clients</groupId>
-			<artifactId>jedis</artifactId>
-			<version>2.9.0</version>
-			<type>jar</type>
-			<scope>compile</scope>
-		</dependency>
+    <dependency>
+       <groupId>com.arangodb</groupId>
+       <artifactId>arangodb-java-driver</artifactId>
+       <version>4.3.1</version>
+    </dependency>
+
+    <dependency>
+    	<groupId>redis.clients</groupId>
+    	<artifactId>jedis</artifactId>
+    	<version>2.9.0</version>
+    	<type>jar</type>
+    	<scope>compile</scope>
+    </dependency>
+
+    <dependency>
+        <groupId>javax.json</groupId>
```

Comment:
Do we use this dependency?

Reply:
I don't think so now, they used to parse the json objects. I'll clean the dependencies(I skipped them before)